server/index.js

```diff
@@ -252,6 +252,7 @@ app.get('/product/:id/related', async (req, res) => {
       const productChar = response.data.characteristics
 
       relatedProducts.push({
+        id: product.id,
         photo: productStyle[0],
```

Comment:
I have memory issues and forgot to also include the product.id in my server code, whoops.